modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/AbstractCreateIndexCommand.java

```diff
@@ -49,10 +50,22 @@ public abstract class AbstractCreateIndexCommand extends AbstractIndexCommand {
 
     protected final List<String> columns;
 
+    protected final CatalogIndexStatus status;
+
+    protected final boolean isCreatedWithTable;
```

Comment:
Maybe remove the prefix "is"?

Reply:
Can I consider this as an optional suggestion?